### PR TITLE
vtctl ApplySchema accepts 'ALTER VITESS_MIGRATION...' statements

### PR DESCRIPTION
**Pull request by shlomi-noach**
## Description

The following is now supported:
```shell
$ vtctlclient ApplySchema -sql "alter vitess_migration 'dce5f77e_51de_11ec_a049_0a43f95f28a3' cancel" commerce
```
`ApplySchema` now accepts all variants of `ALTER VITESS_MIGRATION...`.

#### Why are we doing this?

We long since wanted to deprecate VExec for OnlineDDL. Right now `vtctl OnlineDDL <command>` heavily relies on wrangler/VExec to get the command to the underlying tablets. However, `ApplySchema` already has the required logic to get in to the `QueryService`. We will reimplement `vtctl OnlineDDL` on top of `ApplySchema` instead of `VExec`.

This PR is a first step towards that re-implementation.

## Related Issue(s)

- Tracking:  https://github.com/vitessio/vitess/issues/6926#issuecomment-817619302
- https://github.com/vitessio/vitess/pull/8971 as a previous attempt, reverted since.
- 
## Checklist
- [ ] Should this PR be backported?
- [ ] Tests were added or are not required
- [ ] Documentation was added or is not required

### CODE REVIEW

File: go/vt/schemamanager/tablet_executor.go
Comment (shlomi-noach) [inline go/vt/schemamanager/tablet_executor.go:254]: The change here is to remove `if exec.ddlStrategySetting.IsSkipTopo() {` -- we've already hard coded that as `true` is previous releases.

File: go/vt/schemamanager/tablet_executor.go
Comment (shlomi-noach) [inline go/vt/schemamanager/tablet_executor.go:270]: The change here is to remove `if exec.ddlStrategySetting.IsSkipTopo() {` -- we've already hard coded that as `true` is previous releases.

And of course the `else` code cannot ever happen, so that's also removed.

File: go/vt/schemamanager/tablet_executor.go
Comment (shlomi-noach) [inline go/vt/schemamanager/tablet_executor.go:267]: The change here is to remove `if exec.ddlStrategySetting.IsSkipTopo() {` -- we've already hard coded that as `true` is previous releases.

And of course the `else` code cannot ever happen, so that's also removed.

File: go/vt/schemamanager/tablet_executor.go
Comment (shlomi-noach) [inline go/vt/schemamanager/tablet_executor.go:270]: Here is the actual change in this PR: direct `AlterMigration` statements to the tablets via `QueryService` (that's what the last `true` argument indicates).

**Comment by shlomi-noach** [comment]
Code commits of this PR were already merged by another PR. All that is left in this PR is release notes.

File: doc/releasenotes/13_0_0_summary.md
Comment (deepthi) [inline doc/releasenotes/13_0_0_summary.md:50]:
```suggestion
### vtctl/vtctlclient ApplySchema: ALTER VITESS_MIGRATION
```

Comment (deepthi) [reply doc/releasenotes/13_0_0_summary.md:50]: Suggestion to keep this consistent with other release notes.

**Review by deepthi — APPROVED** [review]
Can be merged after making the requested change + resolve conflicts.

**Comment by deepthi** [comment]
Since the only change it to one file, I suggest doing either a "squash-merge" or a rebase before merge so that there is only one commit with the release notes changes.

**Comment by shlomi-noach** [comment]
squash-merged.

**Pull request merged**
